app/components/Navigation.js

```diff
@@ -2,9 +2,13 @@ import React, { Component } from 'react';
 import { Nav, Navbar, Button } from 'react-bootstrap';
 import axios from 'axios';
 import { connect } from 'react-redux';
+import NavGuest from './NavGuest';
+import NavUser from './NavUser';
 
 class Navigation extends Component {
 	render() {
+		const { user } = this.props;
+		const links = user.firstName ? <NavUser/> : <NavGuest />;
```

Comment:
I would check this using a switch on the userType. We may want to render a different nav for admins as well who will have a first name. Also we can store other cookies (yikes) that would auto-populate shipping fields for example and we could store a first and last name so theoretically so a Guest could theoretically have a firstName as well.